avformat/avformat.go

```diff
@@ -956,6 +960,18 @@ func (ctx *Context) InterruptBlockingOperation() {
 	ctx.CAVFormatContext.interrupt_callback.opaque = unsafe.Pointer(&data)
 }
 
+func (ctx *Context) GetOutputTimestamp(streamIdx int) (int, int, error) {
+	var dts C.int64_t
+	var wall C.int64_t
```

Comment:
Out of curiosity, what does this var represent?

Reply:
the wall clock timestamp that the output was created I think.